### PR TITLE
fix(pie-monorepo): fix yarn.lock conflict

### PR DESCRIPTION
**Pull request by siggerzz**
## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: apps/pie-docs/test/system/404.spec.js [hunk 1]
```diff
@@ -2,7 +2,11 @@ import { test, expect } from '@playwright/test';
 import { disableCookieBanner } from '../playwright/playwright-helper';
 
 test.describe('PIE - 404 Page - @desktop', () => {
+<<<<<<< HEAD
+    test.beforeEach(async ({ page, context, baseURL }) => {
+=======
     test.beforeEach(async ({ page, baseURL, context }) => {
+>>>>>>> 7bf428922b8975e7a9964cd647c98ee26ae7ed5c
```
Comment (raoufswe) [inline apps/pie-docs/test/system/404.spec.js:9]: ops

File: apps/pie-docs/test/system/404.spec.js [hunk 2]
```diff
@@ -16,7 +20,11 @@ test.describe('PIE - 404 Page - @desktop', () => {
 
         // Act
         await Promise.all([
+<<<<<<< HEAD
+            page.waitForResponse((resp) => resp.url() === baseURL),
+=======
             page.waitForResponse((resp) => resp.status() === 200),
+>>>>>>> 7bf428922b8975e7a9964cd647c98ee26ae7ed5c
```
Comment (raoufswe) [inline apps/pie-docs/test/system/404.spec.js:27]: ops ^ 2

File: apps/pie-docs/test/system/cookie-banner.spec.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
 import { test, expect } from '@playwright/test';
 import { disableCookieBanner } from '../playwright/playwright-helper';
 
+<<<<<<< HEAD
+test.beforeEach(async ({ page   , baseURL }) => {
+=======
 test.beforeEach(async ({ page, baseURL }) => {
+>>>>>>> 2f02874aef761367edc70869b9a208994270c86a
```
Comment (raoufswe) [inline apps/pie-docs/test/system/cookie-banner.spec.js:8]: ops ^ 3

File: apps/pie-docs/test/system/404.spec.js (hunk 1 above)
Comment (siggerzz) [reply apps/pie-docs/test/system/404.spec.js:9]: ugh, Somethings up with my commit history.. 2 secs!

**Review by raoufswe — APPROVED** [review]

**Review by LTurns — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**